modules/nw-ptp-configuring-linuxptp-services-as-grandmaster-clock-dual-nic.adoc

```diff
@@ -4,9 +4,14 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="configuring-linuxptp-services-as-grandmaster-clock-dual-nic_{context}"]
-= Configuring linuxptp services as a grandmaster clock for dual E810 Westport Channel NICs
+= Configuring linuxptp services as a grandmaster clock for dual E810 NICs
 
-You can configure the `linuxptp` services (`ptp4l`, `phc2sys`, `ts2phc`) as grandmaster clock (T-GM) for dual E810 Westport Channel NICs by creating a `PtpConfig` custom resource (CR) that configures the host NICs.
+You can configure the `linuxptp` services (`ptp4l`, `phc2sys`, `ts2phc`) as grandmaster clock (T-GM) for dual E810 NICs by creating a `PtpConfig` custom resource (CR) that configures the host NICs.
+
+You can configure the `linuxptp` services as T-GM for the following dual E810 NICs:
+
+* Westport Channel NICs
```

Comment:
Maybe we can add model here like this: (XXVDA4T)

Reply:
Updated it as follows:

You can configure the `linuxptp` services as T-GM for the following dual NICs:

* Intel E810-XXVDA4T Westport Channel NICs
* Intel E810-CQDA2T Logan Beach NICs